angular-rails-app/db/schema.rb

```diff
@@ -13,286 +13,8 @@
-    t.float    "sanitdistrict", limit: 24
-    t.string   "sanitsub",      limit: 255
-    t.string   "address",       limit: 255
-    t.string   "zonedist1",     limit: 255
-    t.string   "zonedist2",     limit: 255
-    t.string   "zonedist3",     limit: 255
-    t.string   "zonedist4",     limit: 255
-    t.string   "overlay1",      limit: 255
-    t.string   "overlay2",      limit: 255
-    t.string   "spdist1",       limit: 255
-    t.string   "spdist2",       limit: 255
-    t.string   "spdist3",       limit: 255
-    t.string   "ltdheight",     limit: 255
-    t.string   "splitzone",     limit: 255
-    t.string   "bldgclass",     limit: 255
-    t.float    "landuse",       limit: 24
-    t.integer  "easements",     limit: 8
-    t.string   "ownertype",     limit: 255
-    t.string   "ownername",     limit: 255
-    t.integer  "lotarea",       limit: 8
-    t.integer  "bldgarea",      limit: 8
-    t.integer  "comarea",       limit: 8
-    t.integer  "resarea",       limit: 8
-    t.integer  "officearea",    limit: 8
-    t.integer  "retailarea",    limit: 8
-    t.integer  "garagearea",    limit: 8
-    t.integer  "strgearea",     limit: 8
-    t.integer  "factryarea",    limit: 8
-    t.integer  "otherarea",     limit: 8
-    t.integer  "areasource",    limit: 8
-    t.integer  "numbldgs",      limit: 8
-    t.float    "numfloors",     limit: 24
-    t.integer  "unitsres",      limit: 8
-    t.integer  "unitstotal",    limit: 8
-    t.float    "lotfront",      limit: 24
-    t.float    "lotdepth",      limit: 24
-    t.float    "bldgfront",     limit: 24
-    t.float    "bldgdepth",     limit: 24
-    t.string   "ext",           limit: 255
-    t.float    "proxcode",      limit: 24
-    t.string   "irrlotcode",    limit: 255
-    t.float    "lottype",       limit: 24
-    t.float    "bsmtcode",      limit: 24
-    t.integer  "assessland",    limit: 8
-    t.integer  "assesstot",     limit: 8
-    t.integer  "exemptland",    limit: 8
-    t.integer  "exempttot",     limit: 8
-    t.integer  "yearbuilt",     limit: 8
-    t.integer  "yearalter1",    limit: 8
-    t.integer  "yearalter2",    limit: 8
-    t.string   "histdist",      limit: 255
-    t.string   "landmark",      limit: 255
-    t.float    "builtfar",      limit: 24
-    t.float    "residfar",      limit: 24
-    t.float    "commfar",       limit: 24
-    t.float    "facilfar",      limit: 24
-    t.integer  "borocode",      limit: 8
-    t.integer  "bbl",           limit: 8
-    t.integer  "condono",       limit: 8
-    t.integer  "tract2010",     limit: 8
-    t.float    "xcoord",        limit: 24
-    t.float    "ycoord",        limit: 24
-    t.string   "zonemap",       limit: 255
-    t.string   "zmcode",        limit: 255
-    t.string   "sanborn",       limit: 255
-    t.float    "taxmap",        limit: 24
-    t.string   "edesignum",     limit: 255
-    t.float    "appbbl",        limit: 24
-    t.datetime "appdate"
-    t.integer  "plutomapid",    limit: 8
-  end
-
```

Comment:
This was cleanup of tables that the Rails application doesn't have to know about.